lib/features/auth/view/screens/sign_up_screen.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:flutter_shakemywidget/flutter_shakemywidget.dart';
+import 'package:telware_cross_platform/core/theme/palette.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:telware_cross_platform/core/utils.dart';
+import 'package:telware_cross_platform/features/auth/view/widget/auth_input_field.dart';
+import 'package:telware_cross_platform/features/auth/view/widget/title_element.dart';
+import 'package:telware_cross_platform/features/auth/view/widget/circular_button.dart';
+import 'package:telware_cross_platform/core/theme/dimensions.dart';
+import 'package:telware_cross_platform/core/theme/sizes.dart';
+import 'package:vibration/vibration.dart';
+
+class SignUpScreen extends ConsumerStatefulWidget {
```

Comment:
as long as you didn't use 'ref' inside the widget, then use a normal StatefulWidget.
when ref is used, then we will convert it to ConsumerStatefulWidget.